UI/Selenium/Hooks/Hooks.cs

```diff
@@ -36,6 +36,7 @@ public class Hooks
         private static string filePath;
         private static Logger Logger = LogManager.Setup().LoadConfigurationFromAppSettings().GetCurrentClassLogger();
         private static string featureTitle;
+        private static int ImageNumber=0;
```

Comment:
It looks to me as when you go to a new scenario that this image number does not get reset back to 0, not a problem really.